configure

```diff
@@ -1323,6 +1306,74 @@ sub push_libpath {
   push_env_dir($buildEnvRef, $libpathname, $path);
 }
 
+if (exists $opts{'qt'}) {
+  my $qglobal = "QtCore" . $slash . "qglobal.h";
+  my $qt_include;
+  my $qt_bin;
+  my $qt_lib;
+
+  if ($try_to_use_qt_system_pkg) {
+    # Try to use pkg-config to get Qt locations
+    my $mesg = ", stopped\nPlease either install Qt or provide the Qt " .
+      "locations (see docs/qt.md for details).\n";
+    `pkg-config --print-variables Qt5Core`;
+    if ($!) {
+      die "ERROR: Trying to use system Qt package but could not confirm " .
+        "Qt and/or pkg-config exists on the system$mesg" .
```

Comment:
This is being reported as an error on at least some versions of Perl.  The statement seems to end in `.` instead of `;`.